realvnc/configValuesFile_test.go

```diff
@@ -0,0 +1,155 @@
+//go:build !windows
+// +build !windows
+
+package realvnc
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestShouldSkipNonKeyValueLines(t *testing.T) {
+	cases := []struct {
+		name      string
+		inputLine string
+	}{
+		{
+			name:      "comment line",
+			inputLine: "  # this is a comment",
+		},
+		{
+			name:      "rogue line",
+			inputLine: "this is a rogue line",
+		},
+	}
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			skipLine, _, _ := ParseConfigKeyValueLine(tc.inputLine)
+			require.True(t, skipLine)
+		})
+	}
+}
+
+func TestShouldReadValidKeyValueLines(t *testing.T) {
```

Comment:
Is it possible to have comments on value lines (like `name=value # comment`), then I think we should have test cases to make sure that's handled correctly.

Reply:
will ask RealVNC if allowed.

Reply:
RealVNC confirmed that they aren't allowed. so i've added a couple of tests for specific params just to make sure that any strings values received with extra comment text on the line will fail validation.